_posts/2017-07-26-a-web-animations-deep-dive-with-angular.md

```diff
@@ -104,7 +104,7 @@ button.addEventListener('click', () => {
 
 When working with frontend frameworks like Angular it could be a whole new story as some of them handle animations differently.
 
-That doesn't mean it's not using some of the core concepts under the hood. Tthey often come with their own animation system. Take Angular's animation system for example, that is built on top of the Web Animations API (WAAPI). It a fairly new and currently being implemented in Chrome and Firefox. It's goal is to unite CSS, JS and SVG. More specifically, it aims to provide the power of CSS performance, add the benefits and flexibility of JS and SVG as well as leave the fundamental work to the browser without the need for additional dependencies.
+That doesn't mean it's not using some of the core concepts under the hood. They often come with their own animation system. Take Angular's animation system for example, that is built on top of the Web Animations API (WAAPI). It's fairly new and currently being implemented in Chrome and Firefox. It's goal is to unite CSS, JS and SVG. More specifically, it aims to provide the power of CSS performance, add the benefits and flexibility of JS and SVG as well as leave the fundamental work to the browser without the need for additional dependencies.
```

Comment:
I've changed `Tthey often come` to `They often come` and `It a fairly new and currently` to `It's fairly new and currently`